### PR TITLE
Add blogpost for 1.10.8 and 1.10.9

### PR DESCRIPTION
**Pull request by kaxil**

### CODE REVIEW

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Apache Airflow 1.10.8 & 1.10.9"
+linkTitle: "Apache Airflow 1.10.8 & 1.10.9"
+author: "Kaxil Naik"
+twitter: "kaxil"
+github: "kaxil"
+linkedin: "kaxil"
+description: "We are happy to present the new 1.10.8 and 1.10.9 releases of Apache Airflow."
+tags: ["release"]
+date: 2020-02-23
```
Comment (kaxil) [inline landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: We can change the date to the date of the release !

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+**Details**:
+* **PyPI**: [https://pypi.org/project/apache-airflow/1.10.9/](https://pypi.org/project/apache-airflow/1.10.9/)
+* **Docs**: [https://airflow.apache.org/docs/1.10.9/](https://airflow.apache.org/docs/1.10.9/)
+* **Changelog (1.10.8)**: [http://airflow.apache.org/docs/1.10.8/changelog.html#airflow-1-10-8-2020-01-07](http://airflow.apache.org/docs/1.10.8/changelog.html#airflow-1-10-8-2020-01-07)
+* **Changelog (1.10.9)**: [http://airflow.apache.org/docs/1.10.9/changelog.html#airflow-1-10-9-2020-02-10](http://airflow.apache.org/docs/1.10.9/changelog.html#airflow-1-10-9-2020-02-10)
+
+Some of the noteworthy new features (user-facing) are:
+
+- [Add tags to DAGs and use it for filtering in the UI](https://github.com/apache/airflow/pull/6489)
```
Comment (ashb) [inline landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]:
```suggestion
- [Add tags to DAGs and use it for filtering in the UI (RBAC only)](https://github.com/apache/airflow/pull/6489)
```

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- [New Executor: DebugExecutor for Local debugging from your IDE](http://airflow.apache.org/docs/1.10.9/executor/debug.html)
+- [Allow passing conf in "Add DAG Run" view](https://github.com/apache/airflow/pull/7281)
```
Comment (ashb) [inline landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: Is this Add DAG run or is it Trigger dag run?

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+- [Allow dags to run for future execution dates](https://github.com/apache/airflow/pull/7038)
```
Comment (ashb) [inline landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: This is specifically only when schedule_interval=None isn't it?

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+```python
+dag = DAG(
+    dag_id='example_dag_tag',
+    schedule_interval='0 0 * * *',
+    tags=['example']
+)
+```
+
+**Screenshot**:
+![Add filter by DAG tags](https://user-images.githubusercontent.com/5796188/68543702-6a96df80-03c3-11ea-8fd3-45b6d77f8ece.png)
```
Comment (ashb) [inline landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: We should probably pull this screen shot in to the site, and not rely on that image staying around on Github.

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md (hunk 4 above)
Comment (kaxil) [reply landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: Yes

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md (hunk 5 above)
Comment (kaxil) [reply landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: Will do

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md (hunk 3 above)
Comment (kaxil) [reply landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: On Trigger Dag Run  yes but the view is called "Add Dag Run":

image

**Comment by potiuk** [comment]
Love it :)

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md (hunk 3 above)
Comment (ashb) [reply landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: What's the tooltip for the action it's accessible from? Cos that whole screen is new.

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md (hunk 4 above)
Comment (ashb) [reply landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: We should say that here then

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md (hunk 3 above)
Comment (kaxil) [reply landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: image

image

Comment (kaxil) [reply landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: "Browse" -> "DAG Runs" -> "Add a new Record" (+ plus sign) brings this screen

File: landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md (hunk 4 above)
Comment (kaxil) [reply landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md]: Updated

**Pull request merged**